source/algorithms.tex

```diff
@@ -9767,7 +9775,7 @@
 \tcode{result} may be equal to \tcode{first}.
 For the overloads with an \tcode{ExecutionPolicy},
 the ranges \range{first}{last} and \range{result}{result + (last - first)}
-shall not overlap.
+does not overlap.
```

Comment:
This is normative damage: this paragraph imposes a requirement on values provided by the user of the library, so it should use "shall".

Reply:
And "the ranges A and B does not overlap" is incorrect grammar anyway.